Pipfile.lock

```diff
@@ -778,19 +802,11 @@
         },
         "requests": {
             "hashes": [
-                "sha256:6c1246513ecd5ecd4528a0906f910e8f0f9c6b8ec72030dc9fd154dc1a6efd24",
-                "sha256:b8aa58f8cf793ffd8782d3d8cb19e66ef36f7aba4353eec859e74678b01b07a7"
+                "sha256:8e5643905bf20a308e25e4c1dd379117c09000bf8a82ebccc462cfb1b34a16b5",
+                "sha256:f71a09d7feba4a6b64ffd8e9d9bc60f9bf7d7e19fd0e04362acb1cfc2e3d98df"
             ],
             "index": "pypi",
-            "version": "==2.26.0"
-        },
-        "setuptools": {
```

Comment:
I guess we should add setuptools into the Pipfile to avoid losing it?

Reply:
I've added it back to the Pipfile and ran `pipenv install setuptools` but it doesn't seem to be added it back into the .lock file. @emrojo any thoughts?

Reply:
Added it back manually